### ISSUE TITLE
Feature Request: Return number of rows affected by update/delete queries.

### ISSUE THREAD

**Issue by crazyjaco** [wp-cli/db-command#120]
Hello.

I recently used WP-CLI to run some SQL queries. 
After they run, I only got a success or failure message.
If it was a success, I did not receive any feedback about how many rows were affected.
Running the query directly in MySQL, it lets me know how many rows were affected.

It was would nice to have that metric returned or optionally returned.

Not sure if that would be part of the normal output or added as a new switch.

Thank you for your time.

**Comment by schlessera** [comment]
Yes, that makes sense and I think it would be a worthwhile improvement.

Would you be up for creating a pull request for this, @crazyjaco ?

**Comment by crazyjaco** [comment]
I would be interested in that, but I don't have a lot of time to spare in the next week or two. I can look at it after that I think. Would that be alright?

**Comment by schlessera** [comment]
@crazyjaco Yes, sure. Feel free to ping me when you started to work on it and get stuck!

**Comment by lkraav** [comment]
I could've used this today in connection with a large `DELETE` operation https://stackoverflow.com/questions/1318972/deleting-millions-of-rows-in-mysql
